### ISSUE TITLE
[PLL] Plain Language Lens Mockup

### ISSUE THREAD

**Issue by joofio** [Gravitate-Health/end-user-app-WP4#14]

**Comment by joofio** [comment]
connected to #20 
a pre-screen with different buttons for RMM, summary and the actual epi

**Comment by riccardo-leoni-dw** [comment]
We have to discuss further or have some examples of the intended outcome.

**Comment by joofio** [comment]
The need for having a possibility of translation in plain terms. I would suggest a hover / similar to a certian phrase and have a translations in simpler terms.
But the UI i will leave it to you :)

**Comment by joofio** [comment]
Schematic of how i see the plain language (and also pictogram and RMM)

[Apresentação2.pptx](https://github.com/Gravitate-Health/end-user-app-WP4/files/14493043/Apresentacao2.pptx)
